.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: go
+go:
+  - 1.7
+
+sudo: false
+
+before_install:
+  - go get github.com/golang/lint/golint
+  - go get github.com/vbatts/git-validation
+
+install:
+  - wget https://github.com/seccomp/libseccomp/archive/v2.3.1.tar.gz
+  - tar -xzvf v2.3.1.tar.gz
+  - pushd libseccomp-2.3.1 && ./configure --prefix=/usr && make && sudo make install && popd
+
+addons:
+  apt:
+    sources:
+    - lucid
```

Comment:
thanks @cyphar, just trying to figure how to get this working on the Travis CI containerized infra w/o building everything from source :)